cache/cache.go

```diff
@@ -68,13 +70,13 @@ func RebuildCmd(c Cache, src, dst string) (err error) {
 	src = filepath.Clean(src)
 	src, err = filepath.Abs(src)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "absolute path failed for %s", src)
```

Comment:
The latest pull request from [https://github.com/drone-plugins/drone-sftp-cache/commit/026fe8066dd3510e24d3c3265f5f6d531d73bcd0](url) has github.com/pkg/errors so I'll leave as it is.